documentation/modules/new-migrating-virtual-machines-cli.adoc

```diff
@@ -777,15 +777,22 @@ spec:
       name: <storage_map> <7>
       namespace: <namespace>
   preserveStaticIPs: <8>
+  networkNameTemplate: <"net-{{.NetworkIndex}}"> <9>
```

Comment:
```suggestion
  networkNameTemplate: <network_name_template> <9>
```

not sure what the  `<...>` notation indicate, if it means a description of the content then it should be a description, 
`"net-{{.NetworkIndex}}"` is an example, it does not look right to put it in brackets, if brackets mean a description or and example, then it's ok, and we should put brackets on all examples, e.g. 14, 15 and 16

p.s.
same for all the other places that use examples